libraries/Microsoft.Bot.Connector/Authentication/ChannelValidation.cs

```diff
@@ -48,18 +48,19 @@ public static class ChannelValidation
         /// HttpClient is used for making those calls. Those calls generally require TLS connections, which are expensive to
         /// setup and teardown, so a shared HttpClient is recommended.</param>
         /// <param name="channelId">The ID of the channel to validate.</param>
+        /// <param name="authConfig">The optional authentication configuration.</param>
         /// <returns>
         /// A valid ClaimsIdentity.
         /// </returns>
-        public static async Task<ClaimsIdentity> AuthenticateChannelToken(string authHeader, ICredentialProvider credentials, HttpClient httpClient, string channelId)
+        public static async Task<ClaimsIdentity> AuthenticateChannelToken(string authHeader, ICredentialProvider credentials, HttpClient httpClient, string channelId, AuthenticationConfiguration authConfig = null)
         {
```

Comment:
To avoid compat issues, add this as a 2nd member with an overloaded (no default) authConfig parameter. 

Note: I'm *Not Sure*, but I believe this is a compat issue, just like adding a ctor paramater is.

Reply:
It is not a backward compat issue, but it *is* a binary compat issue. I'll add a new overload. For the next minor release, I'd like to collapse them back (and explicitly break binary level compat but continue backwards compat) as otherwise we'll keep inflating our overloads unnecessarily.